### ISSUE TITLE
Bug fix:  The path width changer does not work.

### ISSUE THREAD

**Issue by cessnageek** [cessnageek/python-auto-guidance#4]
The path width changer doesn't work.  This appears to be due to some bug in the redraw logic.

**Comment by cessnageek** [comment]
In progress.  Branch Issue4_Path_Width_Fix.  A fix has been implemented but needs tested.

**Comment by cessnageek** [comment]
The fix has been tested.  Once the branch is merged into master, close this issue.

**Comment by cessnageek** [comment]
Issue4_Path_Width_Fix is the branch that needs merged into master.
